.readthedocs.yml

```diff
@@ -11,7 +11,7 @@ build:
   - graphviz
   os: ubuntu-22.04
   tools:
-    python: '3.11'
+    python: '3.12'
```

Comment:
Not sure, but it seems the `ModuleType` can not recognized when build from py3.11.